### PR TITLE
Create index.markdown

### PR DESCRIPTION
**Pull request by moinshaikh**
Translation of index.markdown file into Hindi (Indian-Hindi) language.

### CODE REVIEW

**Comment by moinshaikh** [comment]
I signed it!

File: src/content/hi/fundamentals/getting-started/principles/index.markdown [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+नक़्शा: साझ/संकुचित पृष्ठों की सूची
```
Comment (petele) [inline src/content/hi/fundamentals/getting-started/principles/index.markdown]: The YAML Front Matter attribute should not be translated. The should look something to the effect of:

```
layout: shared/narrow-pages-list
title: "साइट डिजाइन के सिद्धांतों"
description: "गूगल और AnswerLab की जांच उपयोगकर्ताओं को मोबाइल साइटों की एक विविध सेट के साथ बातचीत कैसे एक अध्ययन किया। लक्ष्य सवाल, 'क्या एक अच्छा मोबाइल साइट बनाता जवाब देने के लिए किया गया था?'"
published_on: 2014-08-08
updated_on: 2015-09-17
order: 5
translation_priority: 0
authors:
  - petelepage
```

Check out https://developers.google.com/web/resources/translations/ for the full details on how to translate content.

**Comment by petele** [comment]
@moinshaikh - thanks so much for the contribution. I'm psyched to start to get some Hindi translations!! 

There are a few things that need to be addressed in this PR and #3111 and #3112. Can you address those, and then submit a single new PR that contains everything? 

Thanks,
PEte

File: src/content/hi/fundamentals/getting-started/principles/index.markdown [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-वर्णन: इसका उद्देश था इस प्रश्न का उत्तर निकालना, 'क्या है जो एक अच्छी मोबाइल साइट बनता है ?'
-पर प्रकाशित: २०१४-०८-०८
-को अपडेट किया: २०१५-०९-१७  
-अनुक्रम: ५ 
-अनुवाद प्राथमिकता : ०
-लेखक:
+layout: साझ/संकुचित पृष्ठों की सूची
+title: "साइट डिजाइन के सिद्धांत"
+description: इसका उद्देश था इस प्रश्न का उत्तर निकालना, 'क्या है जो एक अच्छी मोबाइल साइट बनता है ?'
+published_on: २०१४-०८-०८
```
Comment (petele) [inline src/content/hi/fundamentals/getting-started/principles/index.markdown]: Dates, order and translation priority should not be translated, they're data used by the build process.

File: src/content/hi/fundamentals/getting-started/principles/index.markdown [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
+updated_on: २०१५-०९-१७  
+order: ५ 
+translation_priority : ०
+author:
   - petelepage
```
Comment (petele) [inline src/content/hi/fundamentals/getting-started/principles/index.markdown]: Don't forget to add yourself to the contributors as a translator.

**Comment by petele** [comment]
Hey @moinshaikh - thanks for updating, there are still a few outstanding issues that need to be addressed.

**Comment by moinshaikh** [comment]
Hi @petele I would be obliged if you can help me know exact outstanding issues so I can fix them to make the contribution successful. Thanks!

**Comment by moinshaikh** [comment]
@petele I think I have addressed the outstanding issues as mentioned in #3111 and #3112 
Can you pls look into them and advise if any further outstanding issues still need to be addressed.

Thanks,
Moin

**Comment by petele** [comment]
@moinshaikh - will do, need to take care of some other high pri things today, will look Friday.

**Comment by moinshaikh** [comment]
No problem at all! Many thanks for reply. I will now start contributing more Hindi language pages.

Thank you :)
Moin

**Comment by petele** [comment]
@moinshaikh - I'm so sorry, I was just looking at our internal infrastructure and it looks like we don't support Hindi right now. I've filed a bug internally to see if we can get it added. In the mean time, I'll close this and will re-open it if support changes. Sorry.

**Comment by moinshaikh** [comment]
Hi Pete,

Thanks for updating me, I wish it goes through as Hindi is one of the most
widely used language in India/Asia. I look forward to keep contributing to
Google Web Fundamental.

Pls. keep me update if this goes through.


Once again thank you, it really feels great to work with best minds of the
web :)


Moin

**Comment by petele** [comment]
It looks like it didn't take much to get it working on DevSite! 

I've created #3639 to track the work that needs to be done, and PR #3638 to merge it all in. In the mean time, if you want to add more content, do it as a pull request into the hindi-content branch.

Thanks!
PEte

**Comment by petele** [comment]
And we're live! https://developers.google.com/web/fundamentals/getting-started/principles/?hl=hi

**Pull request closed without merging**